include/swift/AST/Module.h

```diff
@@ -789,13 +789,13 @@ class ModuleDecl
   enum class ImportFilterKind {
     /// Include imports declared with `@_exported`.
     Exported = 1 << 0,
-    /// Include "regular" imports with no special annotation.
+    /// Include "regular" imports with an access-level of `public`.
     Default = 1 << 1,
     /// Include imports declared with `@_implementationOnly` or with an
     /// access-level of `package` or less.
     ImplementationOnly = 1 << 2,
-    /// Include imports of SPIs declared with `@_spi`.
-    SPIAccessControl = 1 << 3,
```

Comment:
Shouldn't `@_spi import` still be allowed?

Reply:
We still support `@_spi(X) import` but we didn't really need that information in the filter. That filter is mostly used to call `getImportedModules` which allows mostly to get all public imports, all implementation-only imports, etc. In comparison, the logics specific to `@_spi` use some other filter and then checks for the `spiGroups` field in the returned `AttributedImport`.

Reply:
gotcha sounds good